webpack.config.production.js

```diff
@@ -1,14 +1,24 @@
 var webpack = require('webpack');
+var InlineManifestWebpackPlugin = require('inline-manifest-webpack-plugin');
 var config = require('./webpack.config.base');
 
 config.bail = true;
 config.profile = false;
 config.devtool = 'source-map';
+config.output.filename = '[name].[chunkhash].js';
 
 config.plugins = config.plugins.concat([
   new webpack.LoaderOptionsPlugin({
     debug: true
   }),
+  new webpack.optimize.CommonsChunkPlugin({
+    name: 'vendor',
+    minChunks: (module) => {
```

Comment:
You might consider using a regular function expression here since I don't think you want to bind `this` to `window`. Doesn't make a difference at the moment though